doc/modules/ensemble.rst

```diff
@@ -838,7 +818,135 @@ accessed via the ``feature_importances_`` property::
+One exception is the ``max_iter`` parameter that replaces ``n_estimators``, and
+controls the number of iterations of the boosting process:
+
+  >>> from sklearn.experimental import enable_hist_gradient_boosting
+  >>> from sklearn.ensemble import HistGradientBoostingClassifier
+  >>> from sklearn.datasets import make_hastie_10_2
+
+  >>> X, y = make_hastie_10_2(random_state=0)
+  >>> X_train, X_test = X[:2000], X[2000:]
+  >>> y_train, y_test = y[:2000], y[2000:]
+  >>> clf = HistGradientBoostingClassifier(max_iter=100).fit(X_train, y_train)
+
+  >>> clf.score(X_test, y_test)
+  0.8998
+
+The size of the trees can be controlled through the ``max_leaf_nodes``,
+``max_depth``, and ``min_samples_leaf`` parameters.
+
+The number of bins used to bin the data is controlled with the ``max_bins``
+parameter. Using less bins acts as a form of regularization. It is
+generally recommended to use as many bins as possible, which is the default:
+255 bins for non-missing values.
+
+The ``l2_regularization`` parameter is a regularizer on the loss function and
+corresponds to :math:`\lambda` in equation (2) of [XGBoost]_.
+
+Note that **early-stopping is enabled by default**. The early-stopping
+behaviour is controlled via the ``scoring``, ``validation_fraction``,
+``n_iter_no_change``, and ``tol`` parameters. It is possible to early-stop
+using an arbitrary :term:`scorer`, or just the training or validation loss. By
+default, early-stopping is performed using the the default :term:`scorer` of
+the estimator on a validation set.
+
+Missing values support
+----------------------
+
+:class:`HistGradientBoostingClassifier` and
+:class:`HistGradientBoostingRegressor` have built-in support for missing
+values (NaNs).
+
+During training, the tree grower learns at each split point whether samples
+with missing values should go to the left or right child, based on the
+potential gain. When predicting, samples with missing values are assigned to
+the left or right child consequently.
+
+
+.. TODO: Add this example when missing values PR is merged (results are
+.. wrong for now)
+
+.. from sklearn.experimental import enable_hist_gradient_boosting
+.. from sklearn.ensemble import HistGradientBoostingRegressor
+.. import numpy as np
+
+.. X = np.array([0, 1, 2, np.nan]).reshape(-1, 1)
+.. y = [0, 0, 1, 1]
+.. gbdt = HistGradientBoostingRegressor().fit(X, y)
+.. gbdt.predict(X)
+
+If no missing values were encountered for a given feature during training,
+then samples with missing values are mapped to whichever child has the most
+samples.
+
+
+Low-level parallelism
+---------------------
+
+:class:`HistGradientBoostingClassifier` and
+:class:`HistGradientBoostingRegressor` have parallel implementations that
+use OpenMP through Cython. The number of threads that is used can be changed
+using the ``OMP_NUM_THREADS`` environment variable. By default, all available
+cores are used. Please refer to the OpenMP documentation for details. We are
```

Comment:
maybe a link to openmp's docs?

Reply:
it's tricky, the openmp doc is a huge pdf. I think users are better off just googling `openmp OMP_NUM_THREADS` (first links are from intel and ibm but since these aren't official I'd rather not link to them?)